.taskcat.yml

```diff
@@ -77,21 +77,19 @@ tests:
       # WAF configuration
 
       Section8aInstallWafApp: "Yes"
-      Section8bCreateDeliveryStream: "No" # why delivery stream is same as enabling waf?
-      Section8cWafCreateBucket: "Yes"
-      # Section8dWafLogsBucketName: "" Required if Section8cWafCreateBucket is set to No
-      Section8eWafCreateS3Source: "Yes"
-      Section8fWafBucketPathExpression: "WAF_LOGS/*"
-      Section8gWafLogsSourceCategoryName: "aws/quickstart/waf/logs"
+      Section8bWafCreateBucket: "Yes"
+      # Section8bWafCreateBucket: "" Required if Section8bWafCreateBucket is set to No
```

Comment:
It should be # Section8dWafLogsBucketName: "" Required if Section8bWafCreateBucket is set to No